Utilities/GenerateSecurityGraphs/GenerateSecurityGraphs/Program.cs

```diff
@@ -44,28 +45,16 @@ internal class Options
         [Option('f', "force", Default = false, HelpText = "Create a folder at that path if one doesn't already exist.")]
         public bool Force { get; set; }
 
-        [Option('d', "database", Default = "EdFi_Security", HelpText = "The name of the database containing the authorization metadata.")]
-        public string Database { get; set; }
-
-        [Option(
-            's', "server", Default = "(local)", HelpText = "The name of the SQL Server where the authorization metadata database is located.")]
-        public string Server { get; set; }
-
-        [Option(
-            'u', "user", HelpText =
-                "The SQL Server username to use for connecting to the authorization metadata database.  Leave username and password blank to use integrated security.")]
-        public string User { get; set; }
-
-        [Option(
-            'p', "password", HelpText =
-                "The password to use for connecting to the authorization metadata database.  Leave username and password blank to use integrated security.")]
-        public string Password { get; set; }
-
         [Option(
             'g', "graphviz", Default = @"C:\Program Files\Graphviz\", HelpText =
                 "Graphviz installation path.")]
         public string GraphvizPath { get; set; }
 
+        [Option(
+            'c', "connectionString", Required = true,
```

Comment:
@simpat-jesus could you set a default value and remove required, as done here https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/v7.2/Utilities/GenerateSecurityGraphs/GenerateSecurityGraphs/Program.cs#L50